Formula/p/px.rb

```diff
@@ -3,16 +3,14 @@ class Px < Formula
 
   desc "Ps and top for human beings (px / ptop)"
   homepage "https://github.com/walles/px"
-  url "https://github.com/walles/px.git",
-      tag:      "3.6.13",
-      revision: "e6bd4779970fc448cabb5b714cb737005d6c4318"
+  url "https://files.pythonhosted.org/packages/6c/84/0d8c82c54f5014340df9e44d89e0c09a9261a61bfce656f0a7f09eb3c275/pxpx-3.7.0.tar.gz"
+  sha256 "18961d377c41f65fa944b5c08db6f60a5919c15dbcd83df9f8e95ff594241cb0"
```

Comment:
@branchvincent do you know whether this change will require changes to `px` deployment CI as well?

https://github.com/walles/px/blob/e6f3d9a4961a13cd2271dd1a648c9dba1f43940a/.github/workflows/deployment.yml#L13-L20

```
  homebrew:
    runs-on: ubuntu-latest
    steps:
      - name: Bump Homebrew formula
        uses: dawidd6/action-homebrew-bump-formula@v3
        with:
          token: ${{secrets.JOHAN_GITHUB_API_TOKEN}}
          formula: px
```

"Don't know" is a valid answer.

Reply:
But feel free to ping me if you see otherwise

Reply:
Confirming it worked just fine without any changes, as you predicted: https://github.com/Homebrew/homebrew-core/pull/235577/files

Thanks @branchvincent! :)

Reply:
Awesome glad it worked!